### DISCUSSION TITLE
Deploy

### DISCUSSION THREAD

**Discussion by brunopedrosobuilders** [KaotoIO/kaoto-ui#1136] [Q&A]
I'm trying setup kaoto-backend and kaoto-ui in a kubernetes cluster but front end can't start. 

I think that some configuration is wrong because the api call has prefix /api and backend don't have it.

**UI**
![image](https://user-images.githubusercontent.com/108498424/213620409-bb05557b-5848-4285-820d-445cbf4fcd12.png)

**Backend**
http://kaoto.backend.local/v1/capabilities/

`{"dsls":[{"output":"true","input":"true","deployable":"true","name":"Integration","description":"An Integration defines a workflow of actions and steps.","step-kinds":"[CAMEL-CONNECTOR, EIP, EIP-BRANCH]"},{"output":"true","input":"true","deployable":"false","name":"Camel Route","description":"A camel route is a non deployable in cluster workflow of actions and steps.","step-kinds":"[CAMEL-CONNECTOR, EIP, EIP-BRANCH]"},{"output":"true","input":"true","deployable":"true","name":"Kamelet","description":"A Kamelet is a snippet of a route. It defines meta building blocks or steps that can be reused on integrations.","step-kinds":"[CAMEL-CONNECTOR, EIP, EIP-BRANCH]"},{"output":"true","input":"true","default":"true","deployable":"true","name":"KameletBinding","description":"Kamelet Bindings are used to create simple integrations that link a start step to an end step with optional intermediate action steps.","step-kinds":"[KAMELET, KNATIVE, EIP]"}]}
`

**Comment by mmelko** [comment]
Hi @brunopedrosobuilders ,

How did you install it? just manually deployed the images ? or did you use our yaml template(it's probably not up to date). I guess you are trying to run it on a plain kubernetes right?  

Quickest solution would be to modify your backend ingress to be available at `http://kaoto.backend.local/api`.
